app/components/icons/__test__/Google.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+
+import { Google } from '../Google';
+
+describe('Google', () => {
+  it('should render Google svg icon', () => {
+    const { container } = render(<Google />);
+    expect(container).toBeInTheDocument();
+  });
+});
```

Comment:
I don't think testing svgs would add any value, since they are static assets

Reply:
We are not testing SVG as an individual element. We are wrapping it as a component where the props can change.